bot/cogs/source.py

```diff
@@ -18,42 +19,44 @@ def __init__(self, bot: commands.Bot) -> None:
 
     @in_channel(constants.Channels.bot)
     @commands.command(name="source")
-    async def command_source(self, ctx: commands.Context, command_name: str = None, sub_command: str = None) -> None:
+    async def command_source(self, ctx: commands.Context, *, command_name: str = None) -> None:
         """View the source of a command."""
         if command_name is None:
             await ctx.send("> https://github.com/python-discord/bot")
             return
 
-        if sub_command:
-            await ctx.send(sub_command)
-            command = self.bot.get_command(f"{command_name} {sub_command}")
-        else:
-            command = self.bot.get_command(command_name)
-
+        command = self.bot.get_command(command_name)
         if command is None:
             await ctx.send("No such command found.")
             return
 
-        url = self.get_command_url(command)
+        if command.name == "help":
+            url = "https://github.com/python-discord/bot/blob/master/bot/cogs/help.py#L475-L490"
```

Comment:
In my opinion, we would better send a link without the lines here.

Reply:
Which lines

Reply:
The `#L475-L490` should not be hard-coded in my opinion. I think we should just return the link to the `help.py` file.

Reply:
I think we should just return the correct lines, but without hardcoding them.

@RohanJnr we can't have any hardcoded edge cases in this solution. Please try to find a solution that solves this without hardcoding.